eos/b-decays/lambdab-to-lambdac-l-nu.cc

```diff
@@ -21,17 +21,17 @@
 #include <eos/form-factors/baryonic.hh>
 #include <eos/rare-b-decays/charm-loops.hh>
 #include <eos/b-decays/lambdab-to-lambdac-l-nu.hh>
-#include <eos/utils/complex.hh>
+#include <eos/maths/complex.hh>
```

Comment:
I know it is a bit annoying to do, but I would like to have the header in alphabetic order and classified by sub-directories...

Reply:
Done.